template/sample/MauiSample.csproj

```diff
@@ -69,4 +69,12 @@
         <ProjectReference Include="..\android\NewBinding.Android.Binding\NewBinding.Android.Binding.csproj" />
     </ItemGroup>
 
+	<!-- Reference the Android binding dependencies -->
+	<!-- Uncomment the code block below and update the AndroidLibrary path to point your dependency .aar -->
+	<!-- <ItemGroup Condition="$(TargetFramework.Contains('android'))">
+        <AndroidLibrary Include="..\android\native\newbinding\build\outputs\deps\{yourDependencyLibrary.aar}">

```

Comment:
With the `${buildDir}` update mentioned above we should be able to update this suggested path to

`.\android\native\newbinding\bin\Release\net8.0-android\outputs\deps\{yourDependencyLibrary.aar}`